src/orochi-network/api-reference.md

```diff
@@ -0,0 +1,230 @@
+# API Reference
+
+## Initial
+
+We assumed that you have an initialized instance of Orand
+
+```ts
+import { orand } from "@orochi-network/sdk";
+
+const orandInstance = new orand.Orand({
+  url: "https://orand-test-service.orochi.network/",
+  user: "YOUR_REGISTERED_USERNAME",
+  secretKey: "YOUR_REGISTERED_HMAC_SECRET",
+  consumerAddress: "YOUR_APPLICATION_SMART_CONTRACT_ADDRESS",
+  chaiId: 1,
+});
+```
+
+**Note:**
+
+To prevent replay attack, we're required you to input `consumerAddress`. It's smart contract address that will consume the randomness. Each `consumerAddress` will have different epoch target, you only able to to submit the epoch which has the matched epoch with target epoch on `OrandProviderV1` smart contract.
```

Comment:
```suggestion
To prevent replay attack, we're required you to input `consumerAddress`. It's a smart contract address that will consume the randomness. Each `consumerAddress` will have different epoch target, you only able to to submit the epoch which has the matched epoch with target epoch on `OrandProviderV1` smart contract.
```